x/docs/md/cli/linux.md

```diff
@@ -68,5 +70,20 @@ exercism configure --dir=~/some/other/place
 ### Continue
 You can now continue by [choosing a language](http://exercism.io/languages).
 
+### Uninstalling the Exercism CLI
+
+Remove the files from the `~/bin` directory with the following command
+
+```bash
+rm -rf ~/bin/exercism
+```
+
+You can also remove the `~/bin` directory from your `PATH` only if you have NOT
+installed anything else in that directory. You can do that by opening up your
+`~/.bashrc` file with your favorite editor and remove the `bin` directory that you
```

Comment:
The `-rf` should be removed as well.  If it is a folder and requires the recursive and the force, then we are not quite sure what it is they have done.